spec/system/signup/index.spec.rb

```diff
@@ -0,0 +1,127 @@
+require 'rails_helper'
+
+RSpec.describe 'Signup feature' do
+	it 'should create new user successfully' do
+		visit new_user_registration_path
+		expect(page).to have_content('Sign up new account')
+
+		fill_in('user_full_name', with: 'John Doe')
+		select "ASHA", :from => "user_role"
+		fill_in('user_email', with: 'johndoe@example.com')
+		fill_in('user_phone', with: '9988776677')
+		fill_in('user_password', with: 'JohnDoe')
```

Comment:
We can fill these info using [`Faker`](https://github.com/faker-ruby/faker) for generating names, emails etc. Its like chance.js and we can avoid these hardcoded data.
`Faker` creates different names, emails etc i think. everytime it runs

Reply:
Okay. Will use that.

Reply:
@abhinandanarya06 We also need to store the names and password as well as we need the same for Login as well

Reply:
Then I think we have `create user` command in factorybot. We can use it in Login testing.

Or we can use rspec fixtures if we need to store and reuse in different tests.